src/allotropy/parsers/example_weyland_yutani/example_weyland_yutani_parser.py

```diff
@@ -0,0 +1,89 @@
+from io import IOBase
+import uuid
+
+from allotropy.allotrope.allotrope import AllotropyError
+from allotropy.allotrope.models.fluorescence_benchling_2023_09_fluorescence import (
+    ContainerType,
+    DeviceControlAggregateDocument,
+    DeviceControlDocumentItem,
+    DeviceSystemDocument,
+    MeasurementAggregateDocument,
+    MeasurementDocumentItem,
+    Model,
+)
+from allotropy.allotrope.models.shared.components.plate_reader import (
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueNumber,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import TDateTimeValue
+from allotropy.parsers.example_weyland_yutani.example_weyland_yutani_structure import (
+    Data,
+)
+from allotropy.parsers.lines_reader import CsvReader
+from allotropy.parsers.vendor_parser import VendorParser
+
+
+class ExampleWeylandYutaniParser(VendorParser):
+    def _parse(self, raw_contents: IOBase, _: str) -> Model:
+        reader = CsvReader(raw_contents)
+        return self._get_model(Data.create(reader))
+
+    def _get_model(self, data: Data) -> Model:
+        if data.number_of_wells is None:
```

Comment:
maybe extract `data_number_of_wells` to a local var and use it here and below to be more explicit that it has been checked?